spot_wrapper/wrapper.py

```diff
@@ -1541,6 +1541,49 @@ def list_graph(self, upload_path):
             )
         ]
 
+    def clear_graph(self) -> typing.Tuple[bool, str]:
+        """Clear a graph in a robot.
```

Comment:
This could be clearer, e.g. the docstring for _clear_graph is `Clear the state of the map on the robot, removing all waypoints and edges.`

Reply:
Updated the method docstring. Also added that it removes snapshots only in the ram.